server/services/offenderService.test.ts

```diff
@@ -133,7 +137,7 @@ describe('getOffenders', () => {
         { id: 4, userDescription: 'place 4' },
       ])
 
-      const result = await service.getIncidentLocations(token, 'WRI')
+      const result = await locationService.getIncidentLocations(token, 'WRI')
```

Comment:
these tests are now in the wrong file - this is testing the location service not the offender service

Reply:
You are correct.
Have moved these tests to locationService.test